docker/docker-compose.yaml

```diff
@@ -361,17 +361,18 @@ services:
 
   # The DifySandbox
   sandbox:
-    image: langgenius/dify-sandbox:0.1.0
+    image: langgenius/dify-sandbox:0.2.0
     restart: always
-    cap_add:
-    # Why is sys_admin permission needed?
-    # https://docs.dify.ai/getting-started/install-self-hosted/install-faq#id-16.-why-is-sys_admin-permission-needed
-      - SYS_ADMIN
     environment:
       # The DifySandbox configurations
+      # Make sure you are changing this key for your deployment with a strong key.
+      # You can generate a strong key using `openssl rand -base64 42`.
       API_KEY: dify-sandbox
-      GIN_MODE: release
+      GIN_MODE: 'release'
       WORKER_TIMEOUT: 15
+      ENABLE_NETWORK: 'true'
+    volumes:
+      - ./volumes/sandbox/dependencies:/dependencies
```

Comment:
For what reason we are mounting `python-requirements.txt` in `/dependencies` when launching the sandbox service in runtime, rather than pre-installed ?
Is this a whitelist for allowed dependencies which could be later installed on demand?
If true, what happened to the sandbox container running in offline environment , by design?

Reply:
1. It's a whitelist, all dependencies will be installed while installing, because we are not sure about what dependencies user want, just open it.
2. Using mount to original dependencies path like `site-packages` and `dist-packages` will work for offline environment.